packages/application-components/src/components/maintenance-page-layout/maintenance-page-layout.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Constraints, Text, Spacings } from '@commercetools-frontend/ui-kit';
+import deprecateComponent from '../../utils/deprecate-component';
+import styles from './maintenance-page-layout.mod.css';
+
+const MaintenancePageLayout = props => (
+  <div className={styles.container}>
```

Comment:
This renders a css grid. We can't use the ui-kit Grid as we need additional styles (e.g. `height`)